archlinux/PKGBUILD

```diff
@@ -0,0 +1,15 @@
+# Maintainer: Augusto Hack <hack.augusto@gmail.com>
+pkgname=solc-static
+pkgver=0.4.1
+pkgrel=1
+pkgdesc='solc compiler statically linked against musl'
+arch=('x86_64')
+url='https://github.com/brainbot-com/solidity-static'
+license=('GPL3')
+conflicts=('ethereum' 'solc' 'ethereum-git')
+source=('https://github.com/brainbot-com/solidity-static/releases/download/v0.4.1/solc')
```

Comment:
okay. I think `bumpversion` might be a nice tool here to change the ~3 places for each version raise